eva_submission/eload_validation.py

```diff
@@ -215,6 +228,18 @@ def parse_vcf_check_report(self, vcf_check_report):
                         error_list.append(line.strip())
         return valid, error_list, error_count, warning_count
 
+    def parse_bcftools_norm_report(self, norm_report):
+        total = split = realigned = skipped = 0
+        error_list = []
+        with open(norm_report) as open_file:
+            for line in open_file:
+                if line.startswith('Lines   total/split/realigned/skipped:'):
```

Comment:
Not sure if it is possible to generate a structured report from "bcftools norm". If that is not possible, suggest using a regex here to possibly guard against any changes to the message text...

Reply:
unfortunately no. The inline reporting is the only thing we get from `bcftools norm`.